.github/workflows/test.yml

```diff
@@ -23,17 +23,22 @@ on:
 
 jobs:
   ci:
-    runs-on: <image-name>
+    name: V exercise test on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
 
-      - name: Use <setup tooling>
-        uses: <action to setup tooling>
+      - name: Setup Vlang
+        uses: vlang/setup-v@v1.1
 
-      - name: Install project dependencies
-        run: <install dependencies>
+      # - name: Install project dependencies
+      #   run: <install dependencies>
 
```

Comment:
I thought about deleting but decided to keep them just in case there are dependencies in the future, makes it easier to edit to find where they should go. Maybe that's trying to code for features that don't exist yet, but I think comments fall under a different category. Thoughts?